src/frolladaptive.c

```diff
@@ -24,7 +24,7 @@ void fadaptiverollmean(unsigned int algo, double *x, uint_fast64_t nx, double_an
  */
 
 void fadaptiverollmeanFast(double *x, uint_fast64_t nx, double_ans_t *ans, int *k, double fill, bool narm, int hasna, bool verbose) {
-  if (verbose) Rprintf("%s: running for input length %lu, hasna %d, narm %d\n", __func__, nx, hasna, (int) narm);
+  if (verbose) Rprintf("%s: running for input length %llu, hasna %d, narm %d\n", __func__, nx, hasna, (int) narm);
```

Comment:
just for future readers, `%llu` format works fine for `uint_fast64_t` in `Rprintf` but when using C `sprintf` we need to cast `uint_fast64_t` to `long long unsigned`, otherwise compiler warning will be raised.